packages/aws-cdk-lib/aws-elasticloadbalancingv2-targets/lib/alb-target.ts

```diff
@@ -43,3 +51,37 @@ export class AlbTarget extends AlbArnTarget {
     super(alb.loadBalancerArn, port);
   }
 }
+
+/**
+ * A single Application Load Balancer's listener as the target for load balancing.
+ */
+export class AlbListenerTarget extends AlbArnTarget {
+  /**
+   * Create a new ALB target.
+   * The associated target group will automatically have a dependency added
+   * against the ALB's listener.
+   *
+   * @param albListener The application load balancer listener to target.
+   */
+  constructor(private albListener: elbv2.ApplicationListener) {
+    super(albListener.loadBalancer.loadBalancerArn, albListener.port);
```

Comment:
(memo) using `albListener.port` here seems a right thing to do. (previously it was not validated at all that a corresponding listener to the port exists) 

> the Application Load Balancer you choose from the list must have a listener on the same port as the target group you’re creating.
> https://docs.aws.amazon.com/elasticloadbalancing/latest/network/application-load-balancer-target.html